lib/core/src/server/core-preset-webpack-custom.js

```diff
@@ -12,24 +12,39 @@ function informAboutCustomConfig(defaultConfigName) {
   logger.info(`=> Using default webpack setup based on "${defaultConfigName}".`);
 }
 
-export function webpack(config, { configDir, configType, defaultConfigName }) {
+function wrapPresets(presets) {
+  return {
+    webpackFinal: async (config, args) => presets.apply('webpackFinal', config, args),
+  };
+}
+
+async function createFinalDefaultConfig(presets, config, options) {
   const defaultConfig = createDefaultWebpackConfig(config);
+  return presets.webpackFinal(defaultConfig, options);
```

Comment:
`webpackFinal` will be called twice. One for the `defaultConfig` and second for the `baseConfig`. That is how it worked before the `presets` feature 🤷‍♂️